### DISCUSSION TITLE
How to Achieve on-ice feature in Airflow. Please suggest any way.

### DISCUSSION THREAD

**Discussion by amitsundas09** [apache/airflow#22617] [Ideas]
### Description

let's suppose we have three dags, I want to complete and run success the first dags, but my second dag is failed and is incomplete, I want to skip the second dag and run the third Dag until it is completely successful.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by abdulazim3778-stack** [comment]
Did you find anything how did you implimented the in airflow as u telling the autosys job can be run in the airflow Ui
